Only in ../typo3_src-3.7.0.orig: ImageMagick4
diff -r -u ./t3lib/config_default.php ../typo3_src-3.7.0.orig/t3lib/config_default.php
--- ./t3lib/config_default.php	2004-09-24 07:06:11.000000000 -0400
+++ ../typo3_src-3.7.0.orig/t3lib/config_default.php	2004-11-06 20:57:21.589310840 -0500
@@ -55,8 +55,8 @@
 		'recursiveDomainSearch' => 0,			// Boolean. If set, the search for domain records will be done recursively by stripping parts of the host name off until a matching domain record is found.
 		'devIPmask' => '192.168.*,127.0.0.1',	// Defines a list of IP addresses which will allow development-output to display. The debug() function will use this as a filter. See the function t3lib_div::cmpIP() for details on syntax. Setting this to blank value will deny all. Setting to '*' will allow all.
 		'enable_DLOG' => FALSE,					// Whether the developer log is enabled. See constant "TYPO3_DLOG"
-		'ddmmyy' => 'd-m-y',					// Format of Date-Month-Year - see php-function date()
-		'hhmm' => 'H:i',						// Format of Hours-minutes - see php-function date()
+		'ddmmyy' => "m/d/Y", // Format of Date-Month-Year - see php-function date()
+		'hhmm' => "h:ia", // Format of Hours-minutes - see php-function date()
 		'loginCopyrightWarrantyProvider' => '',		// String: If you provide warranty for TYPO3 to your customers insert you (company) name here. It will appear in the login-dialog as the warranty provider. (You must also set URL below).
 		'loginCopyrightWarrantyURL' => '',		// String: Add the URL where you explain the extend of the warranty you provide. This URL is displayed in the login dialog as the place where people can learn more about the conditions of your warranty. Must be set (more than 10 chars) in addition with the 'loginCopyrightWarrantyProvider' message.
 		'loginCopyrightShowVersion' => 0,		// Boolean: If set, the current TYPO3 version is shown.
diff -r -u ./typo3/install/index.php ../typo3_src-3.7.0.orig/typo3/install/index.php
--- ./typo3/install/index.php	2004-09-24 07:06:11.000000000 -0400
+++ ../typo3_src-3.7.0.orig/typo3/install/index.php	2004-11-06 20:57:33.887441240 -0500
@@ -42,7 +42,7 @@
 
 	// This checks for my own IP at home. You can just remove the if-statement.
 if (1==0 || (substr($_SERVER['REMOTE_ADDR'],0,7)!='192.168' && $_SERVER['REMOTE_ADDR']!='127.0.0.1'))		{
-	die("In the source distribution of TYPO3, the install script is disabled by a die() function call.<br/><b>Fix:</b> Open the file typo3/install/index.php and remove/out-comment the line that outputs this message!");
+//	die("In the source distribution of TYPO3, the install script is disabled by a die() function call.<br/><b>Fix:</b> Open the file typo3/install/index.php and remove/out-comment the line that outputs this message!");
 }
 
 
diff -r -u ./typo3/sysext/install/mod/class.tx_install.php ../typo3_src-3.7.0.orig/typo3/sysext/install/mod/class.tx_install.php
--- ./typo3/sysext/install/mod/class.tx_install.php	2004-09-24 07:06:11.000000000 -0400
+++ ../typo3_src-3.7.0.orig/typo3/sysext/install/mod/class.tx_install.php	2004-11-06 20:57:30.000000000 -0500
@@ -792,6 +792,7 @@
 			$paths=array($GLOBALS["TYPO3_CONF_VARS"]["GFX"]["im_path_lzw"], $GLOBALS["TYPO3_CONF_VARS"]["GFX"]["im_path"], "c:\\php\\imagemagick\\", 'c:\\apache\\ImageMagick\\');
 		} else {
 			$paths=array($GLOBALS["TYPO3_CONF_VARS"]["GFX"]["im_path_lzw"], $GLOBALS["TYPO3_CONF_VARS"]["GFX"]["im_path"], "/usr/local/bin/","/usr/bin/","/usr/X11R6/bin/");
+			$paths[]='/usr/lib/typo3/ImageMagick4';
 		}
 
 		asort($paths);
@@ -2990,6 +2991,8 @@
 			clearstatcache();
 			$verifyImg["filesize"] = @filesize($verifyFile);
 
+			$ex_rows.=$this->getTwinImageMessage("", "<b>Your server:</b>", "<b>Reference:</b>");
+			
 			$ex_rows.='<tr>';
 			$ex_rows.='<td>'.$destImgCode.'</td>';
 			$ex_rows.='<td><img src=clear.gif width=30 height=1></td>';
Binary files ./typo3/sysext/install/verify_imgs/install_read_ai.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_ai.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_bmp.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_bmp.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_gif.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_gif.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_jpg.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_jpg.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_pcx.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_pcx.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_pdf.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_pdf.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_png.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_png.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_tga.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_tga.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_read_tif.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_read_tif.jpg differ
Binary files ./typo3/sysext/install/verify_imgs/install_scale_jpg.jpg and ../typo3_src-3.7.0.orig/typo3/sysext/install/verify_imgs/install_scale_jpg.jpg differ
